README.md

```diff
@@ -43,14 +48,24 @@ bash run_pipeline.sh
 
 For exact instructions on how to reproduce our analysis see `run_pipeline.sh`.
 
+## Results
+
+Our _in silico_ deconvolution of CD4+ cells, CD8+ cells, and Macrophages in TCGA
+data matches very closely to immunohistochemistry estimates of the same cell types
+in a separate dataset. The proportions of immune cell infiltrate across subtypes
+corresponds strongly.
+
+![immune deconvolution and IHC](figures/boxplot_validation_TCGA_summary.png?raw=true)
+
+We also observed that high macrophage infiltration was associated with worse outcomes
+in the TCGA dataset. This relationship was strengthed after adjusting for several
+covariates including age, gender, and gene expression based subtype.
+
+![TCGA macrophage survival](figures/TCGA_kaplanmeier_Macrophages.png?raw=true)  
+
+
 ## Contact
 
 For all code related questions, bug reporting, or feature requests please file a
 [GitHub issue](https://github.com/greenelab/GBM_immune_profiles/issues "file an issue")
 
-## Dependencies
-
-All code was performed in R version 3.2.3 and packages were managed by the
-[checkpoint](https://cran.r-project.org/web/packages/checkpoint/index.html "Microsoft checkpoint r package")
```

Comment:
> So are you no longer using checkpoint?

I still am - see [install script](https://github.com/greenelab/gbm_immune_validation/blob/master/install.R). I removed this part of the README because I thought it was redudant.

> If someone rebuilds the image at a later date, could that break things due to different package versions?

I could see it breaking if the base `ubuntu:16:04` version downloads a different R version for whatever reason and the packages I use end up not being compatible for whatever reason (example see hadley/dplyr#1240). I will update to apt-get specific R versions - I think this solves the issues presented above.

> I think he should specify versions

Sounds good - I will update this section in the next push. I usually don't like (aesthetically) adding dependencies to the README but in this case the code base is small enough that it shouldn't overwhelm.